File: DevExpress-Demo/CS/GridDemo.Wpf/Controls/GridShowcaseBrowserModule.cs
using DevExpress.Xpf.DemoBase;

namespace GridDemo {
    public abstract class GridShowcaseBrowserModule : ShowcaseBrowserModule {
        protected virtual bool NeedChangeEditorsTheme { get { return false; } }
        protected virtual void LoadDemoData() {
        }
    }
}

File: DevExpress-Demo/VB/GridDemo.Wpf/Controls/GridShowcaseBrowserModule.vb
Imports DevExpress.Xpf.DemoBase

Namespace GridDemo

    Public MustInherit Class GridShowcaseBrowserModule
        Inherits ShowcaseBrowserModule

        Protected Overridable ReadOnly Property NeedChangeEditorsTheme As Boolean
            Get
                Return False
            End Get
        End Property

        Protected Overridable Sub LoadDemoData()
        End Sub
    End Class
End Namespace
